GoTwitterCloneBackend/GoTwitterCloneApp/application/playerservice/loginplayer.go

```diff
@@ -33,7 +33,7 @@ func LoginPlayer(ctx context.Context, loginReq models.PlayerLoginRequestModel) (
 	}
 
 	tokenBuilder := utils.BasicJwtTokenBuilder{
-		ExpiresAfter: time.Hour,
+		ExpiresAfter: time.Minute * 1,
```

Comment:
doesnt matter that much but try to remember to return settings to "non-testing" variables, like say returning this back to 1 hour once done.